website/content/docs/v0.7/Learn More/talosctl.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "talosctl"
+weight: 7
+---
+
+The `talosctl` tool packs a lot of power into a small package.  It acts as a
+reference implementation for the Talos API, but it also handles a lot of
+conveniences for the use of Talos and its clusters.
+
+## Client Configuration
+
+Talosctl configuration is located in `$XDG_CONFIG_HOME/talos/config.yaml` if
+`$XDG_CONFIG_HOME` is defined.  Otherwise it is in `$HOME/.talos/config`.  The
+location can always be overridden by the `TALOSCONFIG` environment variable or the
+`--talosconfig` parameter.
+
+Like `kubectl`, `talosctl` uses the concept of configuration contexts, so any
+number of Talos clusters can be managed with a single configuration file.
+Unlike `kubectl`, it also comes with some intelligent tooling (inspired by Joey
```

Comment:
`it also comes with some intelligent tooling (inspired by Joey
Espinosa's excellent `kconf` tool) to manage the merging of new contexts into
the config.`

True to some extent ... I the subject of this paragraph seems to be the talos config file, but I don't think we do any kind of merging in that file. A context in the talos config is different enough that we don't have to because a context is completely self-contained (contains api endpoint and credentials) where in the kubeconfig they are referenced. Hope that makes sense.

Reply:
I could also be wrong, been a while since I looked at the config code :smile: .

Reply:
Right and wrong; yes, the contexts are much simpler, but we do merge seamlessly.  Regardless, the kconf bit doesn't really belong here.